includes/compat/class-wc-stripe-subs-compat.php

```diff
@@ -169,6 +169,28 @@ public function process_payment( $order_id, $retry = true, $force_save_source =
 		}
 	}
 
+	/**
+	 * Overloads WC_Stripe_Payment_Gateway::generate_create_intent_request() in order to
+	 * include additional flags, used when setting payment intents up for off-session usage.
```

Comment:
Note: We'll need to add this same code to [the Pre-Orders compatibility class](https://github.com/woocommerce/woocommerce-gateway-stripe/blob/master/includes/compat/class-wc-stripe-pre-orders-compat.php).

Reply:
I'll switch to this immediately after I merge this PR.